defaults/main.yml

```diff
@@ -6,21 +6,20 @@ odoo_role_odoo_group: odoo
 # VirtualEnv vars
 odoo_role_odoo_venv_path: /opt/.odoo_venv
 
-odoo_role_odoo_edition: oca
-
 # Edition vars
+
 # Vars for the Odoo Nightly edition
-# odoo_role_odoo_edition: "odoo"
+odoo_role_download_strategy: tar
 odoo_role_odoo_version: 11.0
-odoo_role_odoo_release: 20170914
+odoo_role_odoo_release: 20190501
 odoo_role_odoo_url: "https://nightly.odoo.com/{{ odoo_role_odoo_version }}/nightly/src/odoo_{{ odoo_role_odoo_version }}.{{ odoo_role_odoo_release }}.tar.gz"
-odoo_role_odoo_download_path: "/tmp/odoo_{{ odoo_role_odoo_version }}.{{ odoo_role_odoo_release }}.tar.gz"
+odoo_role_odoo_download_path: "{{ odoo_role_odoo_path }}/../odoo_releases/odoo_{{ odoo_role_odoo_version }}.{{ odoo_role_odoo_release }}.tar.gz"
 
 # Vars for the OCA/OCB edition
-# odoo_role_odoo_edition: "oca"
+# odoo_role_download_strategy: git
 odoo_role_odoo_git_url: "https://github.com/OCA/OCB.git"
-# OCA/OCB, branch 11.0, June 11 2018
-odoo_role_git_ref: "8ef3986d58a097a04502d9ca1ee0a860d7230723"
+# OCA/OCB, branch 11.0, May 1 2019
+odoo_role_odoo_git_ref: "903fec011d64cd50a4e925d4046757afacf1363a"
```

Comment:
Instead of using a possibly-broken-in-da-future SHA, should we use a branch name as ref?